test/formatters/installers/ktlint.bash

```diff
@@ -0,0 +1,4 @@
+cd /usr/local/bin
```

Comment:
```suggestion
```

Reply:
At least at present, we want to avoid changing the working directory as that will affect any subsequent formatter install scripts (we will have moved out of the temporary directory that is cleaned up automatically).